examples/contrib/run_transfo_xl.py

```diff
@@ -73,9 +73,7 @@ def main():
     # The pre-processing involve computing word frequencies to prepare the Adaptive input and SoftMax
     # and tokenizing the dataset
     # The pre-processed corpus is a convertion (using the conversion script )
-    tokenizer = TransfoXLTokenizer.from_pretrained(args.model_name)
```

Comment:
Looking at the implementation `TransfoXLTokenizer` and its parent `PreTrainedTokenizer`, this doesn't set any global state, so it can be safely removed.